android/sdk/src/main/java/com/taobao/weex/ui/WXRenderManager.java

```diff
@@ -81,6 +91,58 @@ public WXSDKInstance getWXSDKInstance(String instanceId) {
     return statement.getWXSDKInstance();
   }
 
+    public static int getOpenGLRenderLimitValue() {
+      if(mOpenGLRenderLimitValue == 0){
+          int maxsize = 0;
+          if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+              try {
+                  EGLDisplay dpy = EGL14.eglGetDisplay(EGL14.EGL_DEFAULT_DISPLAY);
+                  int[] vers = new int[2];
+                  EGL14.eglInitialize(dpy, vers, 0, vers, 1);
+                  int[] configAttr = {
+                          EGL14.EGL_BUFFER_SIZE, 32,
+                          EGL14.EGL_ALPHA_SIZE, 8,
+                          EGL14.EGL_BLUE_SIZE, 8,
+                          EGL14.EGL_GREEN_SIZE, 8,
+                          EGL14.EGL_RED_SIZE, 8,
+                          EGL14.EGL_RENDERABLE_TYPE, EGL14.EGL_OPENGL_ES2_BIT,
+                          EGL14.EGL_SURFACE_TYPE, EGL14.EGL_WINDOW_BIT,
+                          EGL14.EGL_NONE};
+                  EGLConfig[] configs = new EGLConfig[1];
+                  int[] numConfig = new int[1];
+                  EGL14.eglChooseConfig(dpy, configAttr, 0, configs, 0, configs.length, numConfig, 0);
```

Comment:
if(numConfig[0] ==0){
       // get GL_MAX_TEXTURE_SIZE
    }
    else{
       //There is no OpenGL Context, GLES10.glGetIntegerv(GLES10.GL_MAX_TEXTURE_SIZE...) will never work
     }